docs/basics/ActionsManagers.md

```diff
@@ -0,0 +1,194 @@
+# Actions Managers
+
+>##### Not familiar with Observables/RxJS?
+> redux-observable requires an understanding of Observables with RxJS. If you're new to Reactive Programming with RxJS, head over to [http://reactivex.io/rxjs/](http://reactivex.io/rxjs/) to familiarize yourself first.
+
+An **Actions Manager** (or Manager for short) is the core primitive of redux-observable.
+
+It is a function which takes a stream of actions and returns a stream of actions. **Actions in, actions out.**
+
+You can think of it of having roughly this type signature:
+
+```js
+function (action$: Observable<Action>, store: Store): Observable<Action>;
+```
+
+While you'll most commonly produce actions out in response to some action you received in, keep in mind that's not actually a requirement! Once you're inside your Manager, use any Observable patterns you desire as long as anything output from the outermost stream is an action.
+
+The actions you emit will be immediately dispatched through the normal `store.dispatch()`, so under the hood redux-observable effectively does `manager(actions$, store).subscribe(store.dispatch)`
```

Comment:
IMO here should be written that not filtering initial action, could cause infinite loop, i.e
`const actionManager = (action$) => action$` will cause infinite loop